### DISCUSSION TITLE
Unable to connect Oracle database

### DISCUSSION THREAD

**Discussion by sohail-anwar1** [nodejs/nodejs.org#5629] [Q&A]
I want to connect node js to my Oracle database 19c (19.18.0.0.0) npm version is 9.6.7 and node version is 18.17.0 when I try to connect node js file to db which is accessed via vpn, I am getting following error. I read all documentation but nothing is working. However I successfully connected through SQL Plus. As per https://node-oracledb.readthedocs.io/en/latest/user_guide/connection_handling.html connection string is correct. Remember I want to connect to a remote db via vpn which I can access through SQL*Plus. 

Error: NJS-515: error in Easy Connect connection string: could not resolve hostname: ERPUAT

**Comment by mikeesto** [comment]
Hi @sohail-anwar1 . This is the repo for the nodejs.org website, we're not a technical support channel here.
